java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -56,6 +58,12 @@ public void transaction_builds_protobuf_request() {
                         Info,
                         ArgsArray.newBuilder().addArgs(InfoOptions.Section.EVERYTHING.toString()).build()));
 
+        transaction.configRewrite();
```

Comment:
Should we add these to ClusterTransactionTests too?

Reply:
Added in 29cdfa49